gensim/test/test_fasttext.py

```diff
@@ -674,7 +674,6 @@ def online_sanity(self, model):
                 terro.append(l)
             else:
                 others.append(l)
-        self.assertTrue(all(['terrorism' not in l for l in others]))
```

Comment:
Why this change?

Reply:
I thought that this line does not check anything worthwhile. By code it is clear that this check is always true. It's not critical for me, so I return this line back.